doc/crypto/BIO_read.pod

```diff
@@ -31,7 +42,10 @@ BIO_puts() attempts to write a NUL-terminated string B<buf> to BIO B<b>.
 
 =head1 RETURN VALUES
 
-All these functions return either the amount of data successfully read or
+BIO_read_ex() and BIO_write_ex() return 1 if data was successfully read or
+written, and 0 otherwise.
```

Comment:
Errrr....  seeing commit 26f321440c707d89cf97903668060fca637ed44d, this can't be true.